src/Microsoft.CodeAnalysis.Analyzers/Core/MetaAnalyzers/Helpers/DiagnosticWellKnownNames.cs

```diff
@@ -6,6 +6,8 @@ namespace Microsoft.CodeAnalysis.Analyzers.MetaAnalyzers.Helpers
 {
     internal static class DiagnosticWellKnownNames
     {
+        internal const string IsKindName = "IsKind";
```

Comment:
`IsKind` isn't available as a language-agnostic API. So it's not available to use here.